app/models/admin_session.rb

```diff
@@ -2,8 +2,17 @@
 
 class AdminSession < DfeSignIn::AuthenticatedSession
   SERVICE_OPERATOR_DFE_SIGN_IN_ROLE_CODE = "teacher_payments_access"
+  SUPPORT_USER_DFE_SIGN_IN_ROLE_CODE = "teacher_payments_support"
```

Comment:
Can we call them "support agent" instead of "support user" to be clear about their role?